src/content/cloud/okteto-cli.mdx

```diff
@@ -17,6 +20,97 @@ Learn more about the Okteto CLI
 - [FAQs](reference/faqs.mdx)
 - [Known Issues](reference/known-issues.mdx)
 
+## okteto up
+
+The `okteto up` command is one of Okteto CLI's main features and key to the Okteto development experience. This command uses the [Okteto Manifest](/reference/manifest) you define to build, deploy, and synchronize your local files with the remote development container.
+
+This command has two modes: `sync` (default mode) and `hybrid` which are detailed below.
+
+|                   | `sync` (default) | `hybrid` |
+| :---------------- | :--------------- | :------- |
+| Source code       | Source code is edited locally and synchronizes bidirectionally (via Syncthing) with a copy of the source code in the remote development container. | Source code is edited locally for a chosen set of microservices. |
+| App location      | The application exists entirely within the remote development container. | Your application is split between your local environment and the remote development container based on your configuration. |
+| `command` context | CLI commands run in the remote development container. | CLI commands run on your local machine, so traffic need to be rerouted to/from the cluster, e.g. using the reverse option |
+
+Below is a reference to help you configure your [`dev` section](/reference/manifest/#dev-object-optional) based on the mode you select. **Standard** configuration options work in both modes and **Sync** options work *only* when `sync` is selected.
+
+For example, if you select `hyrid` mode, you can only use the configuration options in your manifest `dev` section listed under **Standard**. Any configuration options in the **Sync** list will *not work*.
```

Comment:
IMO it seems strange to list under standard the hybrid mode (considering that it is not the default mode). Shouldn't we refer to the modes here and in the section below as `hybrid` or `sync` under this context to avoid confusion?

Reply:
I would avoid saying things like "will not work", I'd say "some configuration options do not apply an will be ignore", or something along those line.

The thing is, all that sounds like we don't want to support it or like if it is unfinished, but the reality is that some options don't make sense given the nature of the selected "mode".

On the other hand, there is a use case that a user might want to switch from one mode to another (maybe testing what mode works best for them or just because it is better suited for an specific case). In that case, they don't need to remove all those options that don't apply in their manifest, those not supported will just be ignored.

Hope this help to explain better the nuances of both modes.

Reply:
@omnomagonz I also don't get what exactly "Standard" means?

Reply:
The reason I landed on Standard vs. Sync (which I agree doesn't sound right in hindsight) was because it didn't make sense to list: Standard (works for both), sync, and hybrid (duplicate of what's listed in Standard).

One option is to list Sync vs. Hybrid and just list all compatible commands even if it means duplicating some.

And I'll make some adjustments to language to avoid "will not work" 👍